testdata/foundry.toml

```diff
@@ -1,14 +1,60 @@
 [profile.default]
-src = "./"
+src = "src"
 test = "./"
-solc = "0.8.18"
-evm_version = "paris"
-ffi = false
-fs_permissions = [{ access = "read-write", path = "./" }]
+optimizer = true
+optimizer_runs = 200
+via_ir = false
+ignored_error_codes = [
+    1878, # SPDX license identifier not provided
```

Comment:
should we close https://github.com/foundry-rs/foundry/issues/10577 with this PR? cc @zerosnacks 
And probably https://github.com/foundry-rs/foundry/pull/10744 not needed anymore

Reply:
it would be cleaner to resolve them instead of ignoring them but I can quite easily create a follow-up PR to reduce the number of ignored error codes, especially `2319` and `2519` need to be addressed

Reply:
#10744 has been closed, I'll update the description of #10577 after merging of this PR and assign it to me